.github/workflows/parachain.yml

```diff
@@ -15,6 +15,32 @@ on:
       - '!parachain/LICENSE'
 
 jobs:
+  fmt:
```

Comment:
I created a new job here because `rustfmt` requires the `nightly` toolchain, while the check toolchain uses the `stable` toolchain.